packages/opds-web-client/src/components/__tests__/Book-test.tsx

```diff
@@ -108,6 +110,23 @@ describe("Book", () => {
       let authors = bookInfo.find(".authors");
       expect(authors.text()).to.equal(bookCopy.contributors[0]);
     });
+
+    it("renders two icons and labels in the compact and expanded views", () => {
+      let wrapper = shallow(
+        <Book
+          book={book}
+          updateBook={updateBook}
+          fulfillBook={fulfillBook}
+          indirectFulfillBook={indirectFulfillBook}
+        />
+      );
+
+      let itemIcon = wrapper.find(".item-icon");
+      let svg = itemIcon.find(AudioHeadphoneIcon);
+
+      expect(svg.length).to.equal(2);
+      expect(itemIcon.first().render().text()).to.equal("NYPL Audio/Headphone SVG Icon Audio");
```

Comment:
I didn't realize the dgx icons said "NYPL" in them. That could be misleading for an OPDS feed that has nothing to do with NYPL.